src/transformers/models/tapas/modeling_tapas.py

```diff
@@ -184,7 +186,7 @@ def load_tf_weights_in_tapas(model, config, tf_checkpoint_path):
                 logger.info(f"Skipping {'/'.join(name)}")
                 continue
         # if first scope name starts with "bert", change it to "tapas"
-        if name[0] == "bert":
+        if name[0] == "bert" or name[0] == "bert_1":
```

Comment:
I'd like to know why this part was added. What exactly is `bert_1` here? Is it optional, or important to the retriever?

Reply:
@NielsRogge